docs/commands/ft.tagvals.md

```diff
@@ -4,7 +4,7 @@ This is useful if your tag indexes things like cities, categories, etc.
 
 !!! warning "Limitations"
     There is no paging or sorting, the tags are not alphabetically sorted.
-    This command only operates on [Tag fields](/redisearch/reference/tags).
+    This command only operates on [Tag fields](/docs/stack/search/reference/tags).
```

Comment:
This should be handled by the doc generator, which properly handles the same link in other docs:
See in
[Tokenization](https://redis.io/docs/stack/search/reference/escaping/#controlling-text-tokenization-and-escaping)
Which properly maps the same link from
```
For tag fields please refer to the [Tag Fields](/redisearch/reference/tags) documentation.
```
And in
[field-types](https://redis.io/commands/ft.create/#field-types)
Which properly maps the same link from
```
For more information, see [Tag Fields](/redisearch/reference/tags).
```

Both are generated properly as pointing to https://redis.io/docs/stack/search/reference/tags
But for `FT.TAGVALS` command, the link is not handled properly by the doc generator.

@itamarhaber seems the fix should not be in this `.md` file